packages/expo-cli/src/commands/eject/Eject.js

```diff
@@ -206,16 +206,19 @@ from \`babel-preset-expo\` to \`babel-preset-react-native-stage-0/decorator-supp
     pkgJson.scripts.ios = 'react-native run-ios';
     pkgJson.scripts.android = 'react-native run-android';
 
-    if (pkgJson.jest && pkgJson.jest.preset === 'jest-expo') {
-      pkgJson.jest.preset = 'react-native';
-      newDevDependencies.push('jest-react-native');
-    } else if (pkgJson.jest) {
-      log(
-        `${chalk.bold(
-          'Warning'
-        )}: it looks like you've changed the Jest preset from jest-expo to ${pkgJson.jest
-          .preset}. We recommend you make sure this Jest preset is compatible with ejected apps.`
-      );
+    if (pkgJson.jest !== undefined) {
+      pkgJson.scripts.test = 'jest';
```

Comment:
If the developer has changed the `test` script from the original `jest` (e.g. added arguments to Jest), I think we shouldn't overwrite it here. Let's just remove this line?

Reply:
Agree, the original author of this line wasn't sure about this either. See https://github.com/react-community/create-react-native-app/pull/473.